openedx/core/djangoapps/schedules/signals.py

```diff
@@ -82,8 +82,8 @@ def reset_schedule_on_mode_change(sender, user, course_key, mode, **kwargs):  #
     # If switching to audit, reset to when the user got access to course. This is for the case where a user
     # upgrades to verified (resetting their date), then later refunds the order and goes back to audit. We want
     # to make sure that audit users go back to their normal audit schedule access.
-    use_availability_date = mode in CourseMode.AUDIT_MODES
-    reset_self_paced_schedule(user, course_key, use_availability_date=use_availability_date)
+    use_enrollment_date = mode in CourseMode.AUDIT_MODES
+    reset_self_paced_schedule(user, course_key, use_enrollment_date=use_enrollment_date)
```

Comment:
I renamed this argument to the method because now regardless of the argument, we consider the course start date. So both paths are kind of an availabililty date. Only difference is if our target date is `now()` or `enrollment.created`. So this name felt better.